build/common/openwisp/settings.py

```diff
@@ -386,3 +387,13 @@
         INSTALLED_APPS.remove('openwisp_monitoring.device')
     if 'openwisp_monitoring.check' in INSTALLED_APPS:
         INSTALLED_APPS.remove('openwisp_monitoring.check')
+
+# For static minification and cache invalidation
+INSTALLED_APPS.append('pipeline')
+# HTML minification with django pipeline
+PIPELINE = { 'PIPELINE_ENABLED': True }
+# static files minification and invalidation with django-compress-staticfiles
+STATICFILES_STORAGE = 'openwisp_utils.storage.CompressStaticFilesStorage'
+# GZIP compression is handled by nginx
+BROTLI_STATIC_COMPRESSION = False
+GZIP_STATIC_COMPRESSION = False
```

Comment:
Hi,

I think this is only applicable of the static files, right?
In that case, it's not applicable for the API, only the dashboard.

Can we move it to `https://github.com/openwisp/docker-openwisp/blob/master/build/openwisp_dashboard/module_settings.py` ?

Reply:
Yes it is only applicable for static files, I think we can move this to `build/openwisp_dashboard/module_settings.py` 👍